### ISSUE TITLE
App cannot start Win10 x64

### ISSUE THREAD

**Issue by robin3010** [Malgardian/TQCollector#9]
Hi! Some problem with new versions of TQC for me. When trying to start app, all I got is two error messages, same for both 1.2.1 and 1.2.2.
![error1](https://user-images.githubusercontent.com/19370567/60280182-16ef9200-990b-11e9-9d94-c88ed07574ab.png)
![error2](https://user-images.githubusercontent.com/19370567/60280192-1ce57300-990b-11e9-8a8b-16ca4cdc7a41.png)
I tried to "unlock" archives after download them and then unpack, clean old versions - had no effect.

**Comment by Malgardian** [comment]
Well, the issue has to be with your config.xml. Did you edit it manually? Did you keep all the files from the zip file in the same folder after unpacking?

**Comment by robin3010** [comment]
Nah, just unpacked and got this error. All files un same folder, of course. :)

**Comment by Malgardian** [comment]
Weird. Well, two things you can do: 1) Post your config.xml here after the error occurred and 2) Try to fill out the config.xml manually with a text editor.

**Comment by robin3010** [comment]
Ok, so:
1) here is my [config.xml](https://github.com/Malgardian/TQCollector/files/3343264/config.zip)
2) that point need some help - which lines should I fill manually besides TQ AE path?

**Comment by Malgardian** [comment]
The AE path and the Vault path. I had a look at your config and it's just fine. Not sure what's happening here. Maybe some antivirus setting interfering?

**Comment by robin3010** [comment]
Ok, so I already tried with no effect:

- fill paths manually
- turn off antivirus (nod32 btw)
- clean all mentions of TQC in registry and reboot
- load in safe mode
- unpack TQC in `C:\Windows\` and run
- all previous version to load

I'm out of options. 😞 

It seems like something interfering, but it's not antivirus, I assume. I set up the antivirus so that it notifies of all interventions in the process at startup, but I only saw the RivaTunerStatisticServer, which I also disabled with no effect.
